coffeescript-loader/loader.js

```diff
@@ -1,6 +1,15 @@
+import { readFile } from 'fs/promises';
 import { readFileSync } from 'fs';
 import { createRequire } from 'module';
-import { URL, pathToFileURL } from 'url';
+import {
+  dirname,
+  resolve as resolvePath,
+} from 'path';
+import {
+  fileURLToPath,
+  pathToFileURL,
+  URL,
+} from 'url';
```

Comment:
path's `resolve` must be renamed or namespaced to avoid conflicting with the resolve hook, which be renamed. I don't have much preference between renaming the import vs namespacing it (the rename seemed DRYer).

👍 for the rest :)

Reply:
> path’s `resolve` must be renamed or namespaced to avoid conflicting with the resolve hook, which be renamed. I don't have much preference between renaming the import vs namespacing it (the rename seemed DRYer).
> 
> 👍 for the rest :)

Ah, good point.